ui/app/components/secure-variable-form.js

```diff
@@ -94,19 +85,29 @@ export default class SecureVariableFormComponent extends Component {
     ]);
   }
 
-  @action
-  validatePath(e) {
-    const value = trimPath([e.target.value]);
+  /**
+   * @typedef {Object} DuplicatePathWarning
+   * @property {string} path
+   */
+
+  /**
+   * @type {DuplicatePathWarning}
+   */
+  @computed('args.{model.path,existingVariables}', 'variableNamespace')
```

Comment:
This `computed` here notifies me of a lurking bug. The reason we're relying on this `computed` instead of reactive getter because there's a breaking test is because of how we're handling `this.args.model.path` in the `Input` in our form.

The `@value` property of `Input` creates a bidirectional binding. In our case, passing `@model.path` is causing the user action to directly manipulate this record in the store. And the `mockedModel` value we pass as an argument in the store isn't a reference to the record in the Ember Data store which causes that test to fail, as it should.

But, if we correct that bug we'd just need to do something like:

```
class SecureVariableForm extends Component {
  @tracked path;

  constructor() {
    this.path = path;
  }

  get duplicatePath() {
    ...remainingLogic();
  }
}

---secure-variable-form.hbs---

<Input @value={{this.path}} /> // instead of @model.path which directly mutates the store

```

Reply:
Resolved, thank you!